uikit-dz/uikit-dz/ViewController.swift

```diff
@@ -4,8 +4,32 @@
 import UIKit
 
 final class ViewController: UIViewController {
```

Comment:
```suggestion
final class ViewController: UIViewController {
```
это общее название ViewController